DEFINITION.md

```diff
@@ -0,0 +1,15 @@
+# CNCF Cloud Native Definition #
+
+*Approved by TOC: [TBD]*
+
+Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic
+environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable
+infrastructure, and declarative APIs exemplify this approach.
+
+These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with
```

Comment:
> These techniques

which techniques ?

Reply:
The ones mentioned in the previous sentence. We could edit it further, but our consensus is that this writing is fairly clear, and that making it more wordy would actually hinder usability.

Reply:
I feel line # 2 and 3 are in reverse order.

Reply:
@tamalsaha Thanks. I understand what you are saying. But we tried the other order already -- see previous drafts. It didn't work as well.

The existing definition includes:

Cloud native systems will have the following properties:
(a) Container packaged. ...
(b) Dynamically managed. ...
(c) Micro-services oriented. ...

That's far more prescriptive and restrictive. These are intended as examples only. 

Some earlier drafts removed all the examples, but many reviewers asked for different mechanisms/patterns from that list to be added back. 

Admittedly it's a shorthand for what would otherwise be a much longer explanation, which, again, was attempted in earlier drafts. 

There were actually a few dozen rounds of revisions. Those 11 drafts were just the major changes, and/or iterations started by different people.

The editing feedback rounds are done. They've been ongoing since January. 

This is out for approval. Or not, but it's really much better than the current charter definition:

https://www.cncf.io/about/charter/